### ISSUE TITLE
Can not create `GreedyProjectionTriangulation` object

### ISSUE THREAD

**Issue by ruoyu0088** [davidcaron/pclpy#17]
The following code raises except `TypeError: pclpy.pcl.surface.GreedyProjectionTriangulation.PointNormal: No constructor defined!`

```
from pclpy import pcl

o = pcl.surface.GreedyProjectionTriangulation.PointNormal()
```

**Comment by davidcaron** [comment]
Thanks for reporting, the constructor is skipped for this class for some reason. I will look into it.

**Comment by davidcaron** [comment]
I'll clean some things and include this fix in v0.9.1
